### DISCUSSION TITLE
Welcome to my "Ask me about ..." page

### DISCUSSION THREAD

**Discussion by serpro69** [serpro69/serpro69#1] [Announcements]
## 👋 Welcome!

  I am using Discussions as a place to connect with other members of our community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by kbenge5-svg** [comment]
Thank you for your assistance I hope I did things right please let me know Y ou all are amazing

**Reply by kbenge5-svg** [reply]
I dont know how to gety files to this page i am working from my phone my chromebook wont connect
